### PR TITLE
Remove tests/performance.php file

### PR DESCRIPTION
**Pull request by mvorisek**
https://github.com/doctrine/sql-formatter/blob/0f7b39d5a4/tests/SqlFormatterTest.php#L153-L167
and
https://github.com/doctrine/sql-formatter/blob/0f7b39d5a4/tests/TokenizerTest.php#L152-L211
are much better performance tests as they use much longer SQL.

The `tests/performance.php` file no longer helpful so I propose to remove it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# SqlFormatter
+# Doctrine SqlFormatter
```
Comment (derrabus) [inline README.md]: Unrelated change?

Comment (mvorisek) [reply README.md]: Do you want an extra PR for it? Or should be name this PR better to cover all improvements?

Comment (derrabus) [reply README.md]: I won't review a "various improvements" PR and I think you know that.

Comment (mvorisek) [reply README.md]: So tell me how many PRs you want from the well self contained commits.

Comment (derrabus) [reply README.md]: I don't have time for this, sorry.

Comment (mvorisek) [reply README.md]: With all the honesty to you, there are 6 commits and I belive we do not deserve 6 PRs. So in order to get this PR passing for both of us, I need your input here.

Comment (derrabus) [reply README.md]: And you got your input.

**Comment by derrabus** [comment]
Too many unrelated chages => closing as explained.

**Comment by mvorisek** [comment]
@derrabus Please reopen this PR, I will extract the other changes to keep here only the "Remove tests/performance.php file".

**Comment by mvorisek** [comment]
Thank you and repushed.

**Comment by derrabus** [comment]
I think the original intend behind this script is to have some sort of benchmark that you can run before and after a change to see if there has been an impact on the performance. I personally haven't used that script and as far as I know we don't run the script in CI either.

I'm fine with removing it, but I'd like to hear the opinion of the other maintainers.

**Comment by greg0ire** [comment]
Fine with removing it as well. If we are going to test performance, we should be using phpbench IMO.

**Review by goetas — APPROVED** [review]
